open_alchemy/cli.py

```diff
@@ -0,0 +1,82 @@
+"""Define the CLI module."""
+import argparse
+import logging
+import pathlib
+
+from open_alchemy import PackageFormat
+from open_alchemy import build_json
+from open_alchemy import build_yaml
+
+# Configure the logger.
+logging.basicConfig(format="%(message)s", level=logging.INFO)
+
+
+def main():
+    """Define the CLI entrypoint."""
+    args = build_application_parser()
+    try:
+        args.func(args)
+    except Exception as exc:  # pylint: disable=broad-except
+        logging.error("Cannot perform the operation: %s.", exc)
+
+
+def build_application_parser():
+    """Build the main parser and subparsers."""
+    # Define the top level parser.
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(title="subcommands", help="subcommand help")
+
+    # Define the parser for the "build" subcommand.
+    build_parser = subparsers.add_parser(
+        "build",
+        description="Build a package with the SQLAlchemy models.",
+        help="build a package",
+    )
+    build_parser.add_argument(
+        "specfile", type=str, help="specify the specification file"
+    )
+    build_parser.add_argument("name", type=str, help="specify the package name")
```

Comment:
Should we add an example? e.g. `"specify the package name, e.g. 'my-package'"`

Reply:
I don't think we should add examples in the CLI. It would get cluttered very quick. Besides it wraps at 80 characters. 

Also, looking at other CLIs (e.g. `docker`, `git`), they don't provide examples on the help screen (but they do in the documentation).